### PR TITLE
Fixes #50 - Incorrect encoding for unchecked bytes

### LINKED ISSUE #50: G1Affine - Raw bytes encoding failing
The raw serialization fails for a G1Affine with the following numbers:

```
x: {0x9af1f35780fffb82, 0x557416ceeea5a52f, 0x1e4403e4911a2d97, 0x0b85bfb438316bf2, 0xa3b716c69a9e5a7b, 0x01fe9b8ad976dd39}
y: {0xb4f1cc806acfb4e2, 0x38c28cba4cf600ed, 0x3af1c2f54a01a366, 0x96a75ac708a9eb72, 0x04253bd59228e50d, 0x120114fae4294c21}
```

### PR DESCRIPTION
**Pull request by vlopes11**
Fixes #50

### CODE REVIEW

**Review by ZER0 — CHANGES_REQUESTED** [review]
LGTM but there are some issue in the changelog.
Also, I would add a comment in the code explaining why the `if` was necessary.

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.5.1] - 25-01-21
+### Changed
+- Fixes #50 - Incorrect encoding for unchecked bytes serialization
```
[the remark this answers is not in the document]
Comment (vlopes11) [reply CHANGELOG.md]: Fixed

**Review by ZER0 — APPROVED** [review]
LGTM

**Comment by ZER0** [comment]
Removing @julesdesmit since we weren't able to contact him and this is a reasonable small fix to merge with just one review.

**Pull request merged**